jbpm-installer/src/test/java/org/jbpm/persistence/scripts/quartzmockentities/QrtzBlobTriggers.java

```diff
@@ -16,23 +16,25 @@
 
 package org.jbpm.persistence.scripts.quartzmockentities;
 
+import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.Id;
+import javax.persistence.Lob;
 
-@Entity(name = "qrtz_blob_triggers")
+@Entity(name = "QRTZ_BLOB_TRIGGERS")
```

Comment:
Just wondering if this change makes any difference.

Reply:
No, just for uniformity